tests/plugins/vllm_add_dummy_model/vllm_add_dummy_model/__init__.py

```diff
@@ -0,0 +1,22 @@
+import torch
+
+from vllm import ModelRegistry
+from vllm.model_executor.models.opt import OPTForCausalLM
+from vllm.model_executor.sampling_metadata import SamplingMetadata
+
+
+class MyOPTForCausalLM(OPTForCausalLM):
+
+    def compute_logits(self, hidden_states: torch.Tensor,
+                       sampling_metadata: SamplingMetadata) -> torch.Tensor:
```

Comment:
```suggestion
                       sampling_metadata: SamplingMetadata) -> Optional[torch.Tensor]:
```

Reply:
Should update the type annotation

Reply:
nice catch! fixed in [aa57613](https://github.com/vllm-project/vllm/pull/7426/commits/aa5761336c28c8d36651e087f45b22cbbe167768)